californium-core/src/test/java/org/eclipse/californium/core/test/lockstep/ObserveClientSideTest.java

```diff
@@ -493,7 +494,7 @@ public void testBlockwiseObserveNotInterruptedByOdlerNotificationWithoutBlockwis
 		server.sendResponse(CON, CONTENT).loadToken("OBS").observe(0).mid(++mid).payload(notifyPayload).go();
 		server.expectEmpty(ACK, mid).go();
 		// Check this one is discard.
-		Response response = request.waitForResponse(1000);
+		Response response = request.waitForResponse(150);
```

Comment:
My experience with "small" timeouts and a "undeterministic" CPU allocation on a shared Buildserver are not that good.

Reply:
I know what do you propose ?

Reply:
keep the 1000 (wasting some maven build time :-) ).

FMPOV, the check, if a response is accidentally received with a timeout, which seems to be too short for "shared build system", would not really check/ensure that.

Reply:
That means increase the `blockwise_life_status` to more than 1s and so made all the test more than 1s longer this is really want you want ?

Reply:
Yes.

Alternatively,  I (or you) creates a new branch and configure the unit test to execute that test repeated (using the RepeatingTestRunner) and set the "hudson_debug" job on it. So we will see, if that timing may cause an issue.